lrmd/lrmd.c

```diff
@@ -1654,6 +1673,37 @@ process_lrmd_rsc_exec(crm_client_t * client, uint32_t id, xmlNode * request)
 }
 
 static int
+process_lrmd_alert_exec(crm_client_t * client, uint32_t id, xmlNode * request)
+{
+    lrmd_rsc_t *alert = NULL;
+    lrmd_cmd_t *cmd = NULL;
+    xmlNode *alert_xml = get_xpath_object("//" F_LRMD_RSC, request, LOG_ERR);
+    const char *alert_id = crm_element_value(alert_xml, F_LRMD_RSC_ID);
```

Comment:
Divergence and a judgment of the handling of create_lrmd_cmd() are necessary when I increase F_LRMD_ALERT and F_LRMD_ALERT_ID.
It thinks me to be good for the future to introduce new processing create_alert_cmd().